lib/ansible/modules/network/nxos/nxos_hsrp.py

```diff
@@ -489,7 +489,7 @@ def main():
         if module.check_mode:
             module.exit_json(changed=True, commands=commands)
         else:
-            load_config(module, commands)
+            body = load_config(module, commands)
```

Comment:
I don't think this should ever return anything. Can you verify that this actually solves the issue?